src/OrbitGl/ThreadTrack.cpp

```diff
@@ -417,6 +434,11 @@ void ThreadTrack::DoUpdatePrimitives(Batcher& batcher, TextRenderer& text_render
           DrawTimesliceText(text_renderer, *timer_info, draw_data.track_start_x, pos, size);
         }
         batcher.AddShadedBox(pos, size, draw_data.z, color, std::move(user_data));
+        if (IsInRange(timer_info, app_->GetHighlightedFunctionId(),
+                      draw_data.histogram_selection_range)) {
+          batcher.AddBoxBorder({pos, size, GlCanvas::kZValueBoxBorder}, TimerTrack::kBoxBorderColor,
+                               std::make_shared<NotPickable>());
```

Comment:
I would agree about having the border on top of the box, but we have to be careful to not see only borders. Did you check it how it looks like with the capture I sent you with 38M timers of the same function? I guess when it's printed as a line it wouldn't bother, but I imagine the case of having boxes of only 2 pixels width being mostly under borders:

For example: (B1: pixel with box number 1, B2: pixel with box number 2, B3: pixel with box number 3, W: white pixel because it's a border). 
-- -- B1 B1 B2 B2 B3 B3 -- -- --
will change to:
-- W B1 W W W W B3 W -- -- 

Should we consider adding borders only when a box it's 4 pixels width?

Reply:
"Borderline" case: 
http://screenshot/8dEr9oNuYMxP4p2
"too-zoomed-out" case:
http://screenshot/5f7wbWbq2cbqRKx